README.md

```diff
@@ -141,6 +141,14 @@ The default tag does not contains the latest API version. Please make sure the l
 
 To fix this error. You should add the latest API version swaggers to the default tag or change default tag to reference latest api version tag.
 
+### MULTIPLE_DEFAULT_TAGS
+
+Level: WARNING

```

Comment:
How about set this rule level is error?

Reply:
If it is error, what is the process to review the error and the criteria to approve.

Reply:
If it's error, service team will know it. It's better if service fix it now, but it's acceptable for us if service plan to fix it in next api-version. In this scenario, it can be approved. @josefree to confirm this rule severity level.

Reply:
I wonder the existing error will not be reported, as it will be ignored in diff.

Reply:
It will be reported when default tags changed, I confirmed just now.